sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaFormat.cs

```diff
@@ -40,5 +40,7 @@ public SchemaFormat(string value)
         public override int GetHashCode() => _value?.GetHashCode() ?? 0;
         /// <inheritdoc />
         public override string ToString() => _value;
+
+        internal string ContentType => $"application/json; serialization={_value}";
```

Comment:
Nit - you're allocating a new string on every access this way.  Your IL looks like:

```cil
    .method assembly hidebysig specialname 
        instance string get_ContentType () cil managed 
    {
        .maxstack 8
        ldstr "application/json; serialization="
        ldarg.0
        ldfld string SchemaFormat::_value
        call string [System.Private.CoreLib]System.String::Concat(string, string)
        ret
    }
```

Consider doing this once in the .ctor if it's going to be used a lot.

Reply:
I'll change to use {get;} = 
so that it is cached.

Reply:
actually add to ctor since it can't be statically initialized.